File: lib/hosts/anonfile.js
const fs = require('fs');
const request = require('request');

const url = 'https://anonfile.com/api/upload';

exports.upload = (fileName, filePath) => {
  const fileData = fs.readFileSync(filePath);

  const formData = {
    file: {
      value: fileData,
      options: {
        filename: fileName
      }
    }
  };

  request.post(
    {
      url,
      formData
    },
    (err, httpResponse, body) => {
      if (err) {
        console.log(`Upload failed ${err}`);
      } else {
        const responseData = JSON.parse(body);
        console.log(responseData.data.file.url.short);
      }
    }
  );
};

File: share/hosts/anonfile.py
from requests import post

__url = "https://anonfile.com/api/upload"


def upload(file_path):
    payload = {
        "file": open(file_path, "rb")
    }

    response = post(__url, files=payload)
    try:
        print(response.json()["data"]["file"]["url"]["short"])
    except Exception as e:
        print("Upload to anonfile failed")
        print(e)
        exit(1)
